proc/test_thread_rand.c

```diff
@@ -0,0 +1,108 @@
+/*
+ * Phoenix-RTOS
+ *
+ * phoenix-rtos-tests
```

Comment:
Please put a short description here. What You're testing?

Reply:
Description added. However some of other test files need checking cause they lack short description.